github-page/_guides/migrate-to-8.md

```diff
@@ -0,0 +1,81 @@
+---
+title: "Migrate to 8.0.0"
+classes: wide
+---
+
+These are the changes needed to migrate to version 8 of Typegoose.
+
+## Typings for document creation
+Document creation now has types, so some old code may fail. Note that now, you MUST specify which parameters are optional and which are mandatory in your classes with `?` and `!`. Note that no modifier means that the property will be assumed required.
+
+```ts
+// This is bad - both name and favoriteNumber are supposed to be mandatory, but appear mandatory to TypeScript
```

Comment:
i think they are `not supposed to be mandatory` but are

Reply:
My bad, will fix.